### ISSUE TITLE
False positive indentation message on single statement blocks within for loops 

### ISSUE THREAD

**Issue by mdouglass** [eslint/eslint#3273]
test.js:
```javascript
for (var i = 0; i < 10; ++i)
    if (i == 0)
        process.exit(0);    // this line causes an indentation error
```

.eslintrc
```json
{
    "rules": {
        "indent": [
            2,
            4
        ]
    },
    "env": {
        "node": true
    },
    "extends": "eslint:recommended"
}
```

Result
```
$ eslint test.js 
test.js
  4:9  error  Expected indentation of 4 characters but found 8  indent

✖ 1 problem (1 error, 0 warnings)

$ eslint -v
v1.0.0
```

If curly braces are added either to the outer for-loop or the inner if-statement, the file passes validation without problem.

This is my first day with eslint, so apologies if I'm missing something obvious.

**Comment by mdouglass** [comment]
I believe that the inner statement can be any statement that has a block without braces. for-for, for-while, etc.

**Comment by gyandeeps** [comment]
Duplicate https://github.com/eslint/eslint/issues/3238
It has been fixed but not yet released.

**Comment by mdouglass** [comment]
Darn, tried to search for it before reporting and missed it.  Thanks!
